controllers/tasksController.js

```diff
@@ -0,0 +1,47 @@
+const taskQuery = require('../models/tasks')
```

Comment:
Rename the modal as simply 'tasks'!

Reply:
Did you mean to rename `const taskQuery` to `cont tasks` ?

Reply:
yes

Reply:
This variable name hasn't been changed. Its a reference to the model, not to a query, hence we should not name it as tasksQuery.